src/OpenLoco/src/Vehicles/RoutingManager.h

```diff
@@ -10,6 +10,9 @@
 
 namespace OpenLoco::Vehicles::RoutingManager
 {
+    constexpr uint16_t kAllocatedButFreeRoutingStation = 0xFFFEU; // Indicates that this array is allocated to a vehicle but no station has been set.
+    constexpr uint16_t kRoutingNull = 0xFFFFU;                    // Indicates that this array is allocated to a vehicle but no station has been set.
```

Comment:
comment copy/paste? or genuinely the same comment?

Reply:
Copy paste error